config/test.json

```diff
@@ -49,5 +49,17 @@
   "logging": {
     "level": "error",
     "fmt": "pretty"
-  }
+  },
+  "serviceClients": [
+    {
+      "id": "d23dbf62b82eb04e",
+      "name": "Test Service Client",
+      "scope": "profile:email",
+      "pubkey": {
+        "kty": "RSA",
+        "n":"xaQHsKpu1KSK-YEMoLzZS7Xxciy3fsGrhrrqW_JBrq3IRmeGLaqlE80zcpIVnStyp9tbet2niYTemt8ug591YWO5Y-S0EgQyFTxnGjzNOvAL6Cd2iGie9QeSehfFLNyRPdQiadYw07fw-h5gweMpVJv8nTgS-Bcorlw9JQM6Il1cUpbP0Lt-F_5qrzlaOiTEAAb4JGOusVh0n-MZfKt7w0mikauMH5KfhflwQDn4YTzRkWJzlldXr1Cs0ZkYzOwS4Hcoku7vd6lqCUO0GgZvkuvCFqdVKzpa4CGboNdfIjcGVF4f1CTQaQ0ao51cwLzq1pgi5aWYhVH7lJcm6O_BQw",
+        "e":"AQAB"
```

Comment:
A JWK set makes perfect sense here. Looking at JKUs, and their use in the auth-server, I don't see pros. It looks like the auth-server is fetching the JWK set over HTTPS... that is a route served by the exact same server? How, then, is this decoupled from the config?